diff --git a/compat-openssl11.spec b/compat-openssl11.spec
index 1300256..7fb7dfe 100644
--- a/compat-openssl11.spec
+++ b/compat-openssl11.spec
@@ -111,44 +111,44 @@ releases.
 cp %{SOURCE12} crypto/ec/
 cp %{SOURCE13} test/
 
-%patch1 -p1 -b .build   %{?_rawbuild}
-%patch2 -p1 -b .defaults
-%patch3 -p1 -b .no-html  %{?_rawbuild}
-%patch4 -p1 -b .man-rename
-
-%patch31 -p1 -b .conf-paths
-%patch32 -p1 -b .version-add-engines
-%patch33 -p1 -b .dgst
-%patch36 -p1 -b .no-brainpool
-%patch37 -p1 -b .curves
-%patch38 -p1 -b .no-weak-verify
-%patch40 -p1 -b .disable-ssl3
-%patch41 -p1 -b .system-cipherlist
-%patch42 -p1 -b .fips
-%patch45 -p1 -b .weak-ciphers
-%patch46 -p1 -b .seclevel
-%patch47 -p1 -b .ts-sha256-default
-%patch48 -p1 -b .fips-post-rand
-%patch49 -p1 -b .evp-kdf
-%patch50 -p1 -b .ssh-kdf
-%patch51 -p1 -b .intel-cet
-%patch52 -p1 -b .s390x-update
-%patch53 -p1 -b .crng-test
-%patch55 -p1 -b .arm-update
-%patch56 -p1 -b .s390x-ecc
-%patch60 -p1 -b .krb5-kdf
-%patch61 -p1 -b .edk2-build
-%patch62 -p1 -b .fips-curves
-%patch65 -p1 -b .drbg-selftest
-%patch66 -p1 -b .fips-dh
-%patch67 -p1 -b .kdf-selftest
-%patch69 -p1 -b .alpn-cb
-%patch70 -p1 -b .rewire-fips-drbg
-%patch71 -p1 -b .conf-new
-%patch72 -p1 -b .disable-fips
-%patch73 -p1 -b .cve-2022-0778
+%patch 1 -p1 -b .build   %{?_rawbuild}
+%patch 2 -p1 -b .defaults
+%patch 3 -p1 -b .no-html  %{?_rawbuild}
+%patch 4 -p1 -b .man-rename
+
+%patch 31 -p1 -b .conf-paths
+%patch 32 -p1 -b .version-add-engines
+%patch 33 -p1 -b .dgst
+%patch 36 -p1 -b .no-brainpool
+%patch 37 -p1 -b .curves
+%patch 38 -p1 -b .no-weak-verify
+%patch 40 -p1 -b .disable-ssl3
+%patch 41 -p1 -b .system-cipherlist
+%patch 42 -p1 -b .fips
+%patch 45 -p1 -b .weak-ciphers
+%patch 46 -p1 -b .seclevel
+%patch 47 -p1 -b .ts-sha256-default
+%patch 48 -p1 -b .fips-post-rand
+%patch 49 -p1 -b .evp-kdf
+%patch 50 -p1 -b .ssh-kdf
+%patch 51 -p1 -b .intel-cet
+%patch 52 -p1 -b .s390x-update
+%patch 53 -p1 -b .crng-test
+%patch 55 -p1 -b .arm-update
+%patch 56 -p1 -b .s390x-ecc
+%patch 60 -p1 -b .krb5-kdf
+%patch 61 -p1 -b .edk2-build
+%patch 62 -p1 -b .fips-curves
+%patch 65 -p1 -b .drbg-selftest
+%patch 66 -p1 -b .fips-dh
+%patch 67 -p1 -b .kdf-selftest
+%patch 69 -p1 -b .alpn-cb
+%patch 70 -p1 -b .rewire-fips-drbg
+%patch 71 -p1 -b .conf-new
+%patch 72 -p1 -b .disable-fips
+%patch 73 -p1 -b .cve-2022-0778
 %patch -P 83 -p1 -b .replace-expired-certs
-%patch74 -p1 -b .cve-2023-0286
+%patch 74 -p1 -b .cve-2023-0286
 
 cp apps/openssl.cnf apps/openssl11.cnf
 

